src/web/next.config.js

```diff
@@ -1,21 +1,12 @@
-const path = require('path');
-const dotenv = require('dotenv');
-
 // Forward the API_URL value from the root env to NEXT_PUBLIC_API_URL
-const envPath = path.join(process.cwd(), '../..', '.env');
-if (dotenv.config({ path: envPath }).error) {
-  console.warn('Unable to read root Telescope .env file.');
+if (process.env.API_URL) {
+  // API_URL was already set on the env (likely this is
+  // happening within docker) so use that.
+  process.env.NEXT_PUBLIC_API_URL = process.env.API_URL;
 } else {
-  const apiUrl = process.env.API_URL;
-
-  if (apiUrl) {
-    console.info(`Copying API_URL=${apiUrl} to NEXT_PUBLIC_API_URL`);
-    process.env.NEXT_PUBLIC_API_URL = apiUrl;
-  } else {
-    const port = process.env.PORT || 3000;
-    console.warn(`No API_URL set in .env, using http://localhost:${port}`);
-    process.env.NEXT_PUBLIC_API_URL = `http://localhost:${port}`;
-  }
+  // No env, so likely we're working locally, so make a guess:
+  const port = process.env.PORT || 3000;
+  process.env.NEXT_PUBLIC_API_URL = `http://localhost:${port}`;
```

Comment:
This didn't work. The value of `process.env.API_URL` at build time is `undefined`.